wall_e_install.sh

```diff
@@ -32,15 +33,15 @@ mkdir -p "$HOME/esp"
 pushd "$HOME"/esp || (echo "Error: Cannot Make Directory" && exit 1)
 
 #ESP Toolchain & ESP-IDF
-git clone -b release/v4.2 --recursive https://github.com/espressif/esp-idf.git
+git clone -b v4.4.3 --recursive https://github.com/espressif/esp-idf.git
 pushd esp-idf
 ./install.sh
 popd
 popd
 
 #Check if installation is successfull 
 . $HOME/esp/esp-idf/export.sh 
-idf.py --version | (grep "v4.2" && echo "Installation successfull") \
+idf.py --version | (grep "v4.2" && echo "Installation successfull! Please restart your computer for the changes to take effect.") \
```

Comment:
* shouldn't it be `grep "4.4.3"` ?
* Why is there a need for restart ?

Reply:
In the CI we are using release-v4.4, shouldn't that be used ?

Reply:
@SuperChamp234 remove the usermod part entirely, instead explain to the attendees why giving port access is necessary and why it is needed.

Reply:
> In the CI we are using release-v4.4, shouldn't that be used ?

`release-v4.x.x` means pre-release versions, while cloning the tags like `v4.x.x` represents the released version.
We should update our CI to use the tag `v4.4.3`

Reply:
> > In the CI we are using release-v4.4, shouldn't that be used ?
> 
> `release-v4.x.x` means pre-release versions, while cloning the tags like `v4.x.x` represents the released version. As quoted in esp-idf docs :
> 
> > If a required feature is not yet available in a stable release, but you do not want to use the master branch, it is possible to check out a pre-release version or a release branch.
> 
> We should update our CI to use the tag `v4.4.3`

No, I mean this `git clone -b release/v4.2` -> release/v4.4